pkg/astprinter/astprinter.go

```diff
@@ -93,12 +93,12 @@ func (p *printVisitor) indentationDepth() (depth int) {
 	case ast.NodeKindOperationDefinition,
 		ast.NodeKindFragmentDefinition:
 	default:
-		return 2
+		return countIndentDepth(p.indent)
```

Comment:
Indentation depth is used to determine at what level of nesting we are

You just need to have 1 as a base value instead of 2 as it was before

Old behavior value 2 and indent `.` results in `..` value
With new behavior same could be achieved with value 1 and indent `..`